clients/zctl/zctl.c

```diff
@@ -183,6 +183,7 @@ main(int argc,
 		if (code)
 		    fprintf (stderr, "%s: %s: %s\n",
 			     argv[0], error_message (code), ssline);
+		ZClosePort();
```

Comment:
Why? Is this just to close the UPnP session before we exit?

Reply:
Yes, otherwise the UPnP port mapping will persist on the router until you reboot the router.

Reply:
This is really making me want to have a proper zephyr_ctx that we can manage the lifecycle with. It's a bit annoying we can't easily test for these sorts of port leaks. Not something to address now, of course.